.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+### Pull Request (PR) Title
+
+Things the PR does go here
+- these can be bullet
+- points like this
+- if you like
+
+### Pre-PR Checklist:
+
+Have you
+- [ ] Updated NUbook if necessary (add link to NUbook PR here)
+- [ ] Added/Updated tests for your changes, including regression tests for bug fixes
+- [ ] Updated relevant module READMEs
+
+### Optional Headings (delete this heading and the ones you don't use)
+
+### Reviewers, Note
+
+Things the reviewers should keep in mind go here.
+Also tell reviewers how to verify/test your code's correctness here.
+
+### Things left to do
+
+- [ ] If you have more things to do
+- [ ] Add them here
+
+**If there are many things left to do then consider a [_draft_ PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/)**
+
+## Labels Info (delete the rest of this once you have read it)
```

Comment:
Maybe this stuff can just go into the Pre-PR Checklist in one line, like 
- [ ] [Added relevant labels to the PR](https://github.com/NUbots/NUbots/labels)

Reply:
I like the idea of adding this as a todo item as you say, so I have added that. However, I think it's worth documenting the label system here, because this is the place you're going to be wondering most about how the labels are organised.

Reply:
I think it'd be good to add in a link to all the labels (https://github.com/NUbots/NUbots/labels)